src/test/components/GraphTimeButtons.test.js

```diff
@@ -11,7 +11,12 @@ describe('<GraphTimeButtons>', () => {
     expect(wrapper.find('#Yearly').exists()).toBe(true);
   });
 
-  it('matches snapshot', () => {
-    expect(mount(<GraphTimeButtons />)).toMatchSnapshot();
+  it('renders with provided props', () => {
+    const props = {
+      activeID:   'not sure what this should be',
```

Comment:
iirc, that's supposed to actually be a time interval value - weekly, monthly, or yearly.

Reply:
Whoops. Meant to look that up.